paper-server/src/main/java/org/bukkit/craftbukkit/entity/CraftBee.java

```diff
@@ -118,4 +118,18 @@ public void setTicksSincePollination(int ticks) {
     public int getTicksSincePollination() {
         return this.getHandle().ticksWithoutNectarSinceExitingHive;
     }
+
+    @Override
+    public void setTimeSinceSting(int time) {
+        if (time < 0) {
+            throw new IllegalArgumentException("Time since sting cannot be negative");
```

Comment:
You can use a precondition, can you also rebuild patches to sort the AT file if you don't mind.